src/common-app.js

```diff
@@ -0,0 +1,22 @@
+// TODO: Publish artsy morgan as an npm module.
+import logger from "artsy-morgan"
+import express from "express"
+import commonMiddlewareSetup from './common-middleware'
+
+const app = express()
+
+app.set("trust proxy", true)
+
+app.use(logger)
+
+// Install common middlewares
+commonMiddlewareSetup(app)
+
+// novo must be first, the current app contains a catch all route.
+import novo from "./novo/index"
+app.use("/", novo)
+
+import current from "./current"
```

Comment:
Do these `import`s get hoisted during babel transpilation?

Reply:
It wouldn't make a difference either way, I'm mainly organizing this file for future refactoring.